Doc/library/venv.rst

```diff
@@ -99,6 +122,17 @@ Creating virtual environments
    environment in its new location. Otherwise, software installed into the
    environment may not work as expected.
 
+You can deactivate a virtual environment by typing "deactivate" in your shell.
+The exact mechanism is platform-specific and is an internal implementation
+detail (typically a script or shell function will be used).
+
+
+.. note:: When a virtual environment is active, any options that change the
+   installation path will be ignored from all :mod:`distutils` configuration
+   files to prevent projects being inadvertently installed outside of the
+   virtual environment.
+
+
 .. _venv-api:
```

Comment:
```suggestion
.. _venv-api:
```

Distutils has been officially deprecated since Python 3.10 (the oldest backportable version) and removed in this version, 3.12 (thus the link should not resolve). Furthermore, use of these config files and use of `distutils` as an installer directly has been deprecated for perhaps a decade. Therefore, this should just be removed.

Reply:
The one thing that would make me hesitate with this one is that it would be good to get these improvements into older Python docs. The ``VIRTUAL_ENV`` environment variable documentation is wrong since Python 3.8 (by a commit that was backported that far 2 years ago).

Reply:
But that snippet could be left out even in documentation for older Python versions, as it doesn't really add all that much information - for example, it's not clear which options might be meant in

> any options that change the installation path

Reply:
> The one thing that would make me hesitate with this one is that it would be good to get these improvements into older Python docs. 

Right, which is why I mentioned it has been deprecated since the oldest backportable version, Python 3.10, and discouraged for perhaps a decade (never mind "distutils configuration files"—I don't know if I've ever run into any modern mention of anything actually using them these days; I'm involved with a fair bit of packaging stuff and I'd never even heard of them until now). Ergo, this is likely to cause more confusion than anything, if even packaging experts like @vsajip aren't even sure what it is intended to mean. And if worst comes to worse, this one change can always be dropped in a backport (though it seems very unnecessary).

> The `VIRTUAL_ENV` environment variable documentation is wrong since Python 3.8 (by a commit that was backported that far 2 years ago).

I'm not entirely sure what you intend to mean here, sorry.

Reply:
> I'm not entirely sure what you intend to mean here, sorry.

Simply stating that there is a factually incorrect statement in the documenation relating to ``VIRTUAL_ENV`` environment use (it claims: "This can be used to check if one is running inside a virtual environment.", which is wrong), which it would be great to remove from as many backported versions as possible to avoid incorrect usage in the wild.

Reply:
Ah, gotcha :+1: I was confused and wasn't sure the connection.